### DISCUSSION TITLE
%E7%AB%99%E9%95%BF%E4%BB%8B%E7%BB%8D/%E5%BC%80%E5%8F%91%E6%97%A5%E8%AE%B0

### DISCUSSION THREAD

**Discussion by giscus** [cpython666/cpython666.github.io#4] [Announcements]
# %E7%AB%99%E9%95%BF%E4%BB%8B%E7%BB%8D/%E5%BC%80%E5%8F%91%E6%97%A5%E8%AE%B0

Python斗罗的代码笔记

https://cpython666.github.io/%E7%AB%99%E9%95%BF%E4%BB%8B%E7%BB%8D/%E5%BC%80%E5%8F%91%E6%97%A5%E8%AE%B0.html

**Comment by cpython666** [comment]
太详细了吧，学到了很多🕶️

**Comment by tingshuo-yiqing** [comment]
刚刚就是蚊香图进来的
